petastorm/codecs.py

```diff
@@ -101,6 +101,9 @@ def spark_dtype(self):
 class NdarrayCodec(DataframeColumnCodec):
     """Encodes numpy ndarray into a spark dataframe field"""
 
+    def __init__(self, compression=False):
+        self.compression = compression
```

Comment:
Doc string please.

Reply:
done

Reply:
My question is whether you can read existing datasets with this code (since the signature of the codec has changes). Can you please confirm against msds 17 for example?